### PR TITLE
Test a stale review request

### LINKED ISSUE #68: Test task lists
Lala

### PR DESCRIPTION
**Pull request by petros**
This is a default body.

Fixes https://github.com/githubhelp/playground/issues/68

Fixes #68

### CODE REVIEW

File: hello_world.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def  hello_world(name)
+  puts "Hi #{name}!"
+end
```
Comment (petros) [inline hello_world.rb]: One comment

Comment (mr-chef) [inline hello_world.rb]: One comment

Comment (mr-chef) [inline hello_world.rb]: One comment here

**Review by mr-chef — APPROVED** [review]
👍

**Comment by petros** [comment]
Fixes https://github.com/githubhelp/playground/issues/68

Fixes #68

**Pull request closed without merging**